run_fvs.sh

```diff
@@ -6,17 +6,34 @@ date
 pwd
 git status
 
-./build_node.sh
-./create_binary.sh
+docker rm -f host1 || true
+docker run --privileged -v `pwd`:/code --name host1 -tid jpetazzo/dind
+
+docker exec -t host1 bash -c \
+ 'docker -d & \
```

Comment:
Hmmm, the dind container will already be running a docker daemon so it's a bit strange to run another. I suspect that by running another daemon you're tramping over the the one that's run for you. 

Something like `while ! docker ps; do sleep 1; done` would capture the meaning of 'waiting for docker to start' better than just starting a new daemon.